docs/components/operate/userguide/monitor-operation-status.md

```diff
@@ -0,0 +1,27 @@
+---
+id: monitor-operation-status
+title: Monitor operation status
+description: "Monitor and investigate the status of any operation"
+---
+
+Operate offers a powerful way to monitor and investigate the status of any operation you started. This includes canceling and retrying process instances, process instance migration and modification.
+
+First go to the processes view and expand the operations panel:
+
+![operate-view-process-filters](./img/monitor-operation-state/operations-panel.png)
+
+Select an operation from the list by clicking on the operation id. Notice that the operation if is set as a filter:
```

Comment:
Should we add a note in the case the user doesn't have any operations and sees "No operations have been created yet."?

Reply:
Good idea, I added it.